neo4j-ogm-docs/src/main/asciidoc/reference/nativetypes.adoc

```diff
@@ -0,0 +1,183 @@
+[[reference:native-property-types]]
+= Native property types
+
+Neo4j distinguishes between property, structural and composite types.
+While you can map attributes of Neo4j-OGM entities very easy to composite types
+(please read <<reference:type-conversion:custom:using-composite-types, the example using a custom converter for composite types>> for more information),
+most of the attributes are usually property types.
+
+The most important property types are
+
+* `Number`
+* `String`
+* `Boolean`
+* The spatial type `Point`
+* Temporal types: `Date`, `Time`, `LocalTime`, `DateTime`, `LocalDateTime` and `Duration`
+
+`Number` has two subtypes (`Integer` and `Float`).
+Those are not the Java types with the same name but Neo4j specific types that map to `long` and `double` respectively.
+Please refer to both the https://neo4j.com/docs/cypher-manual/current/syntax/values/[Cypher] and https://neo4j.com/docs/developer-manual/current/drivers/cypher-values/[Java-Driver manual] for further information about the type system.
+
+While you have to take a bit of care when modelling entities with numeric attributes (in regards of precession and scale),
+mapping of numbers, strings and boolean attributes is pretty much straight forward.
+Temporal and spatial types however made their first appearances in Neo4j 3.4.
+Therefore OGM provided <<reference:type-conversion:custom, type conversion>> for those to store them as string or numeric types.
+Especially it maps all temporal types onto ISO 8601 formatted strings and spatial types onto a composite, map based structure.
```

Comment:
Replace `Especially it maps` with `In particular, it mapped`

Reply:
Should be `, it maps`, right?